<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>321</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>CLOKWISE/COUNER</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>ON/OFF MOOTR</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="verticalSlider">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>391</width>
     <height>140</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_6"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEdit_7"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_8"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>290</y>
     <width>232</width>
     <height>132</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBox_5">
      <property name="text">
       <string>ON/OFF Ventilator</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="text">
       <string>Mauel/Auto Grejac</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_4">
      <property name="text">
       <string>ON/OFF Grejac</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_6">
      <property name="text">
       <string>Manuel/Auto Ventilator</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
